crates/scroll/revm/src/states/bundle.rs

```diff
@@ -70,9 +70,19 @@ impl From<(BundleState, &ScrollPostExecutionContext)> for ScrollBundleState {
     }
 }
 
+// This conversion can cause a loss of information since performed without additional context.
+#[cfg(any(not(feature = "scroll"), feature = "test-utils"))]
```

Comment:
we feature gate this lossful conversion to avoid using it in production code with the Scroll feature